source/agora/common/Block.d

```diff
@@ -332,39 +357,50 @@ unittest
     block.txs ~= txs;
     block.header.merkle_root = block.buildMerkleTree();
 
-    const Hash h0 = hashFull(txs[0]);
-    const Hash h1 = hashFull(txs[1]);
-    const Hash h2 = hashFull(txs[2]);
-    const Hash h3 = hashFull(txs[3]);
-    const Hash h4 = hashFull(txs[4]);
-    const Hash h5 = hashFull(txs[5]);
-    const Hash h6 = hashFull(txs[6]);
-    const Hash h7 = hashFull(txs[7]);
+    Hash[] hashes;
+    hashes.length = txs.length;
+    hashes.reserve(txs.length);
```

Comment:
Small suggestion:
```suggestion
```

The `.length` call above it has already allocated the memory space, so this `reserve` call doesn't do anything extra.